wallet/src/components/IbcTransfer.tsx

```diff
@@ -375,9 +456,22 @@ export const IbcTransferInternal = ({
       }
     />
   ) : (
-    <Box sx={{ marginY: 3 }}>
-      <Button onClick={() => connectWithKeplr()} variant="contained">
-        Connect With Keplr
+    <Box sx={{ marginTop: '6px' }}>
+      <Typography fontSize={12} sx={{ color: 'rgba(0,0,0,0.6)' }}>
+        From
+      </Typography>
+      <Button
+        onClick={() => isKeplrConnectionInProgress || connectWithKeplr()}
```

Comment:
This makes the button inert while the flag is true but that's a surprising UX for the click to do nothing. Why not disable it instead?

Reply:
Disabling it makes the color too faint, and overriding the default styling is painful as witnessed in your other comment. I think visually it's unsurprising because of the progress indicator and keplr pop up in this case, but I just made this more clear to the screen reader as well, PTAL.

Reply:
Trying to wrangle this stuff definitely makes me appreciate tailwind more.

Reply:
Very good point, added a comment